website/docs/r/virtual_machine.html.markdown

```diff
@@ -1267,6 +1267,25 @@ The options are:
 
 * `join_domain` - (Optional) The domain name in which to join  the virtual machine. One of this or `workgroup` must be included.
 
+* `domain_ou` - (Optional) The MachineObjectOU which specifies the full LDAP path name of the OU to which the virtual machine belongs.
+
+**Example**:
+
+```hcl
+resource "vsphere_virtual_machine" "vm" {
+  # ... other configuration ...
+  clone {
+    # ... other configuration ...
+    customize {
+      # ... other configuration ...
+      windows_options {
+        domain_ou = "OU=Sandbox,OU=Workstations,DC=example,DC=com"
```

Comment:
```suggestion
        domain_ou = "OU=bar,OU=foo,DC=example,DC=com"
```

Reply:
Resolved in https://github.com/hashicorp/terraform-provider-vsphere/commit/e27590dd9fb5b6dab92e5341c932be6fdd1421ae.